### PR TITLE
Bug report update -- round 2

### PR DESCRIPTION
**Pull request by gante**
# What does this PR do?

The previous PR (#31983) had something wrong in its config file, which made the bug report option disappear. 

Screenshot 2024-07-16 at 17 06 28

This PR adds a markdown section at the top of the bug report page. The markdown section was inspired by the [example](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms) in the GH docs.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.yml [hunk 1]
```diff
@@ -1,6 +1,16 @@
 name: "\U0001F41B Bug Report"
-description: Submit a bug report to help us improve transformers. Be sure to check our [bug report guidelines](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#did-you-find-a-bug) before submitting
+description: Submit a bug report to help us improve transformers
```
Comment (gante) [inline .github/ISSUE_TEMPLATE/bug-report.yml:2]: description just like before the previous PR. I suspect it doesn't accept links 😅

Comment (amyeroberts) [inline .github/ISSUE_TEMPLATE/bug-report.yml:2]: Let's add an automatic label too 🏷️ 

```suggestion
description: Submit a bug report to help us improve transformers
labels: [ "bug" ]
```

**Review by amyeroberts — APPROVED** [review]
Thanks!

File: .github/ISSUE_TEMPLATE/bug-report.yml (hunk 1 above)
Comment (gante) [reply .github/ISSUE_TEMPLATE/bug-report.yml:2]: yes! 💛

**Pull request merged**